Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -608,8 +605,18 @@ await using (jellyfinDb.ConfigureAwait(false))
                 }
             }
 
+            var libraryDb = await Resolve<IDbContextFactory<LibraryDbContext>>().CreateDbContextAsync().ConfigureAwait(false);
+            await using (libraryDb.ConfigureAwait(false))
+            {
+                if ((await libraryDb.Database.GetPendingMigrationsAsync().ConfigureAwait(false)).Any())
+                {
+                    Logger.LogInformation("There are pending EFCore migrations in the database. Applying... (This may take a while, do not stop Jellyfin)");
```

Comment:
Probably want to specify what database the migrations are for since the same line is logged right above